installbuilder/datafiles/Base_DSC.data

```diff
@@ -500,6 +500,7 @@ chown root:root /opt/microsoft/${{SHORT_NAME}}/Scripts/OMSServiceStatAll.sh
 chown root:root /opt/microsoft/${{SHORT_NAME}}/Scripts/OMSYumUpdates.sh
 chown root:root /opt/microsoft/${{SHORT_NAME}}/Scripts/OMSZypperUpdates.sh
 chown root:root /opt/microsoft/${{SHORT_NAME}}/Scripts/OMSAptUpdates.sh
+chown root:root /opt/microsoft/${{SHORT_NAME}}/Scripts/NPMAgentBinaryCap.sh
```

Comment:
At this point, "NPMAgentBinaryCap.sh" script might not have been created as "nxOMSAgentNPMConfig" DSC module is an optional module. So, it would be better if we modify the owner permissions of the above script in "npm_startup.sh" script in which "NPMAgentBinaryCap.sh" creation happens.